runtime/krest/src/lib.rs

```diff
@@ -993,6 +993,8 @@ parameter_types! {
 		inflation_stagnation_rate: Perbill::from_percent(1),
 		inflation_stagnation_year: 10,
 	};
+	pub const InitializeInflationAt: BlockNumber = 0;
```

Comment:
Let us merge it first and I'll check these two configurations

Reply:
Yup, i figured we'll discuss these and implement before any runtime upgrade